packages/administration/src/Component/Config/Action/Builder/Action.php

```diff
@@ -15,6 +15,8 @@ final class Action extends AbstractAction
 {
     private ?ActionRouteInterface $actionRoute = null;
 
+    private bool $openInNewTab = false;
```

Comment:
why is it moved from the abstract class here?

Reply:
Two reasons:

1. AbstractAction knows nothing about links so this setting is just weird. You can set it as open in a new tab, but you can't configure a link
2. I was wondering what would a DropdownAction look like, which would not have any link itself, but only a list of other Actions. And then it doesn't make sense either

Reply:
ok, thanks for the clarification